src/pages/postgraphile/make-wrap-resolvers-plugin.md

```diff
@@ -141,6 +146,21 @@ module.exports = makeWrapResolversPlugin(
 );
 ```
 
+### Usage
+
+As mentioned above, the generated plugin module should then be provided as an argument to `--append-plugins` when using PostGraphile CLI, or included in the `appendPlugins` array option when using PostGraphile as a library.
+
+The above examples defined single plugin object generated by calling `makeWrapResolversPlugin` and exported as the only element in a Javascript file (module). This is equivalent to the following in ES6:
```

Comment:
```suggestion
The above examples defined a single plugin function generated by calling `makeWrapResolversPlugin` and exported via CommonJS as the only element in the JavaScript file (module).

If you are using ES6 modules (`import`/`export`) rather than Common JS (`require`/`exports`), then the syntax should be adjusted slightly:
```